src/java.base/share/classes/java/util/JapaneseImperialCalendar.java

```diff
@@ -1856,6 +1856,13 @@ protected void computeTime() {
 
         if (isSet(ERA)) {
             era = internalGet(ERA);
+            // Don't check under, historically we have allowed values under
+            // BEFORE_MEIJI to be ignored during normalization
+            // We check against eras.length over Reiwa ERA due to possibility
+            // of additional eras via "jdk.calendar.japanese.supplemental.era"
```

Comment:
The possibility is not only that emergency property, but by design Japanese calendar eras increase when the current emperor abdicates.